rflx/pyrflx/typevalue.py

```diff
@@ -492,42 +504,69 @@ class MessageValue(TypeValue):
 
     _type: Message
 
-    def __init__(self, model: Message, refinements: Sequence[Refinement] = None) -> None:
+    def __init__(
+        self,
+        model: Message,
+        refinements: Sequence[Refinement] = None,
+        skip_verification: bool = False,
+        fields: Mapping[str, "MessageValue.Field"] = None,
+        checksums: Mapping[str, "MessageValue.Checksum"] = None,
+        type_literals: Mapping[Name, Expr] = None,
+    ) -> None:
+        # pylint: disable=too-many-arguments
```

Comment:
It would be better to introduce a new class containing `fields`, `checksum` and `type_literals` and pass them as one parameter (similar as in `model.AbstractMessage`). The current implementation gives the impression that it would make sense to set only one of them.

Reply:
While we currently don't use it that way there might be scenarios where it can be useful to e.g. only set the type literals or only the checksums. It is supposed to work at least.

Reply:
I still think it is a bad coding style to have that many arguments. Wouldn't it make sense group them together at least to reduce the number of arguments? You could still make parts of the object optional to allow only setting one of the three if that is needed in the future.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/405/commits/c20876d3b3ea87cd0b4a9b1688f172f0190f907d.